astro-docs/src/content/docs/technologies/dotnet/Guides/migrate-from-nx-dotnet-core.mdoc

```diff
@@ -0,0 +1,448 @@
+---
+title: Migrate from @nx-dotnet/core to @nx/dotnet
+description: Step-by-step guide to migrate from the community @nx-dotnet/core plugin to the official @nx/dotnet plugin
+sidebar:
+  label: Migrate from @nx-dotnet/core
+filter: 'type:Guides'
+---
+
+This guide provides a streamlined migration path from the community `@nx-dotnet/core` plugin to Nx's official `@nx/dotnet` plugin.
+
+{% callout type="note" title="Plugin Status" %}
+The `@nx-dotnet/core` plugin is deprecated in favor of the official `@nx/dotnet` plugin (available in Nx 22+), which is maintained by the Nx team and integrated directly into the Nx repository.
+{% /callout %}
```

Comment:
```suggestion
{% aside type="note" title="Plugin Status" %}
The `@nx-dotnet/core` plugin is deprecated in favor of the official `@nx/dotnet` plugin (available in Nx 22+), which is maintained by the Nx team and integrated directly into the Nx repository.
{% /aside %}

```

Prefer asides since they look more native in the astro docs (I still need to go remove the old callout ones)